### PR TITLE
chore: remove Express restaurant because it's gone in reality

### LINKED ISSUE #110: Kanske ta bort Express??

### PR DESCRIPTION
**Pull request by Jassob**
It's good if mat reflects reality. This commit makes mat more reality reflective and fixes #110.

Thank you Rembane for the above commit message.

### CODE REVIEW

File: test/Main.hs [hunk 1]
```diff
@@ -24,24 +24,6 @@ testFun expected = either
...
-               (T.pack "Vegetariska biffar, persiljekr\228m & potatismos")
-        ]
-
-    $ parse
-        "Swedish"
-        (BL8.pack
-          "{\"data\":{\"dishOccurrencesByTimeRange\":[{\"displayNames\":[{\"name\":\"Kycklinggryta, persilja, senap & kokt potatis\",\"categoryName\":\"Swedish\"},{\"name\":\"Chicken stew, parsley, mustard & boiled potatoes\",\"categoryName\":\"English\"}],\"startDate\":\"9/9/2019 12:00:00 AM\",\"dishType\":{\"name\":\"Express\"},\"dish\":{\"name\":\"Kycklinggryta, aubergin, zucchini, tomat & rostad potatis\"}},{\"displayNames\":[{\"name\":\"Vegetariska biffar, persiljekr\195\164m & potatismos\",\"categoryName\":\"Swedish\"},{\"name\":\"Vegetarian patties, parsley cream & mashed potatoes\",\"categoryName\":\"English\"}],\"startDate\":\"9/9/2019 12:00:00 AM\",\"dishType\":{\"name\":\"Express - Vegan\"},\"dish\":{\"name\":\"Vegetariska biffar, majonn\195\164s & potatispur\195\169\"}}]}}\n"
-        )
-    )
-
```
Comment (The1Penguin) [inline test/Main.hs]: While I agree with the premise of this PR, test for Kåren would still be good, one could convert it for current Kårresturangen

Comment (Jassob) [reply test/Main.hs]: Right, then I'll fix that.

I don't remember enough of this repo to know if Express was part of Kåren or not, but your absolutely right!

Comment (Rembane) [reply test/Main.hs]: I got there first! :D Is fix! 

https://github.com/dtekcth/mat-chalmers/pull/111/commits/41d04a1efd647c1b56a0fff65ba3ec325268f83e

**Review by The1Penguin — APPROVED** [review]
LGTM! :+1:

**Pull request merged**